CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+## Version 1.0.0 2017-01-28
+- Updated return format so that both promises and callback now have the same structure:
+
+```javascript
+{
+  'resp': <req response>,
```

Comment:
I don't think we want to save characters here, so `response` would be a better name.
Also, it's unclear what's `response` and what's `data`.
How about using more descriptive names and maybe adding some examples or describing the shape of those data structures?